src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java

```diff
@@ -137,11 +136,66 @@ private static void createDriver() throws UnsupportedBrowserException, Malformed
         webDriver.get(WEBDRIVER_STARTING_URL);
     }
 
+    private static void configureDriver(BrowserManager webDriverManager, String browserType) {
+        final String noDriverPathWarnMessage = "The value of property 'webdriver.drivers.path is not specified."
+                + " Trying to automatically download and setup driver.";
+
+        if (!WEBDRIVER_PATH.isEmpty()) {
+            System.setProperty("webdriver." + browserType + ".driver", new File(WEBDRIVER_PATH).getAbsolutePath());
+        } else {
+            LOG.warn(noDriverPathWarnMessage, WEBDRIVER_BROWSER_NAME);
```

Comment:
WEBDRIVER_BROWSER_NAME was here because first arg of LOG.warn contained '{}'. Perhaps now it is not needed?